src/Raven.Server/Documents/TcpHandlers/TcpConnectionOptions.cs

```diff
@@ -114,9 +114,10 @@ public void Dispose()
         }
 
 #if !RELEASE
+        private string ss = Environment.StackTrace;
```

Comment:
Can we remove this?

Reply:
Yes, I added this to see if it would fail with additional information.